frontend/src/api/api.ts

```diff
@@ -26,6 +26,14 @@ export async function conversationApi(
           },
           { type: 'text', text: apiMessage.content }
         ]
+      } else if (uploadedFile.type === FileType.Csv) {
```

Comment:
[Pebble] I'm not able to comment on the unchanged above, but I realized as we were testing that we are passing in the uploaded file as part of the message object in the API request, although it's not needed or used.

I think we should remove the `uploaded_file` key from that object so it's not in the API payload (can take up a lot of space for big file). So on Line 13, could you add `delete apiMessage.uploaded_file`. Then, on Line 14, you could pull the key from the original `message` like `message.uploaded_file`. It may make sense to even move that to the top so it's clear we are pulling that key out into a variable, then cloning the object, then removing the key we wanted